### PR TITLE
Add code tabs for _tour/nested-functions

### PR DESCRIPTION
**Pull request by flomebul**

### CODE REVIEW

File: _tour/nested-functions.md [hunk 1]
```diff
@@ -24,10 +27,32 @@ In Scala it is possible to nest method definitions. The following object provide
...
+{% tab 'Scala 3' for=Nested_functions_definition %}
+```scala
+ def factorial(x: Int): Int =
+    def fact(x: Int, accumulator: Int): Int =
+      if x <= 1 then accumulator
+      else fact(x - 1, x * accumulator)
+    fact(x, 1)
+
+ println("Factorial of 2: " + factorial(2))
+ println("Factorial of 3: " + factorial(3))
```
Comment (bishabosha) [inline _tour/nested-functions.md:41]: The indentation is inconsistent here - `fact` is indented 3 spaces from `factorial`, but the `if` is only indented 2 spaces. Also both 2 and 3 code examples seem to have a 1 space margin

Comment (bishabosha) [reply _tour/nested-functions.md:41]: I realise it was already indented this way, but it would be nice to clean it up now

**Review by bishabosha — APPROVED** [review]

**Comment by bishabosha** [comment]
@flomebul not sure what happened here

**Comment by flomebul** [comment]
I wrongly delete some file locally...

**Pull request closed without merging**